content/manager/upload-blueprint.md

```diff
@@ -6,43 +6,42 @@ draft: false
 weight: 400
 ---
 
-Before Cloudify Manager can deploy your blueprint, you must upload it. You can upload a blueprint using the CLI or (for Premium users) the Cloudify Web interface.
+Before you can deploy a blueprint, you must upload the blueprint to the Cloudify Manager. You can upload a blueprint using the CLI. Premium users can also upload using the Cloudiy Web Interface.
 
 Either use a blueprint that you have written or download an [example blueprint](https://github.com/cloudify-cosmo/cloudify-nodecellar-example) to upload.
 
-## Uploading via the CLI
 
-From the Cloudify command-line interface, you can upload your blueprint to Cloudify Manager. You must specify a path to a blueprint file. Cloudify compresses the folder and its contents.
+## Uploading a Blueprint using the Cloudify Web Interface
 
-The following is an example of `upload`:
-{{< gsHighlight  bash >}}
-$ cfy blueprints upload -b BLUEPRINT_ID -p BLUEPRINT_FILE_LOCATION
-...
-
-...
-{{< /gsHighlight >}}
-
-
-## Uploading a Blueprint via the Cloudify Web Interface
-
-If you are a Premium version user, you can upload a pre-packaged blueprint archive, such as *.tar, *.tar.gz, *.tar.bz, *.zip., using the Cloudify Manager UI.
+If you are a Premium version user, you can upload a pre-packaged blueprint archive using the Cloudify Web Interfacein tar, tar.gz, tar.bz, or zip formats.
 
 1. On the **Blueprints** widget, click **Upload**.   
    ![The blueprint upload button]({{< img "manager/ui_upload_blueprint_button.png" >}})
-2. In the Upload blueprint dialog, either specify the URL of the blueprint archive, or select it from the filesystem.  
+2. In the Upload blueprint dialog, either enter the URL of the blueprint archive, or select the file from the filesystem.
    ![The blueprint upload dialog]({{< img "manager/ui-upload-blueprint.png" >}})
-3. Specify a distinguishing name for the blueprint.   
-   For example, you might want to specify one instance of the blueprint upload as `blueprint-template` and another as a `blueprint-with-input`.
-4. (Optional) Specify the YAML filename.   
-   This field refers to to the *.yaml file that contains the application topology. If left blank, the default `blueprint.yaml` file is used. 
+3. Enter a unique name for the blueprint.   
+   For example, you can upload one instance of the blueprint as `blueprint-template` and another instance as a `blueprint-with-input`.
+4. (Optional) Select the YAML filename of the blueprint.
+   This field refers to the .yaml file that contains the application topology. If left blank, the default `blueprint.yaml` file is used. 
```

Comment:
I'd add - in case this field is left blank, and no file named blueprint.yaml exists in the archive, an error will occur.